hikari/api/rest.py

```diff
@@ -1289,6 +1284,11 @@ def fetch_reactions_for_emoji(
             If the channel or message is not found.
         hikari.errors.InternalServerError
             If an internal error occurs on Discord while handling the request.
+
+        !!! note
+            The exceptions on this endpoint will only be raised once the
+            result is awaited or interacted with. Invoking this function
```

Comment:
might be better to say "iterated over" here as "interacting" with an object may just mean looking at attributes